.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Notes:
+* Pull requests will not be accepted until the submitter has agreed to the [contributer agreement](https://github.com/ARMmbed/mbed-os/blob/master/CONTRIBUTING.md).
+* This is just a template, so feel free to use/remove the unnecessary things
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+
+## Migrations
+If this PR changes any APIs or behaviors, give a short description of what "API users" should do when this PR is merged.
```

Comment:
Why "API users"? Is it a nickname? If you just mean to draw attention to this, then use * or **